services/testspace/testspace-test-count.service.js

```diff
@@ -0,0 +1,69 @@
+'use strict'
+
+const { metric: metricCount } = require('../text-formatters')
+const TestspaceBase = require('./testspace-base')
+
+module.exports = class TestspaceTestCount extends TestspaceBase {
+  static route = {
+    base: 'testspace',
+    pattern:
+      ':metric(total|passed|failed|skipped|errored)/:org/:project/:space+',
```

Comment:
space will often be specified with a branch name, and of course branches in source repos can have `/`